woocommerce/woocommerce-android
# Purchase button adjustments

Part of: #12281

### Description
This PR includes the purchase button below the shipment details bottom sheet and improves the landscape support for the main shipping label screen by including the mark order complete control. The changes on this PR are only focused on the UI. Additionally, it updates the mark order complete caption as recommended by design.

The UI adjustments were discussed here: p1732055886685969-slack-C02KUCFCSFP

### Testing information

1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap con create shipping labels
5. Chek that the purchase button is displayed on the main screen
6. check that the purchase button is displayed after expanding the shipment details section
7. Change the phone configuration to landscape
8. Check that on the landscape, the mark order complete switch is displayed 
9. Check that expanding/collapsing the shipment details section works as expected on landscape. 

### The tests that have been performed
- Checking the UI is displayed as expected

### Images/gif

https://github.com/user-attachments/assets/7a02c070-040f-4c43-a47d-417981f66199



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
